examples/tutorials/velocity_control_sim.ipynb

```diff
@@ -0,0 +1,274 @@
+{
```

Comment:
What does this do? Can you add a markdown cell that describes what this is and what it does ?

Reply:
@exhaustin

Reply:
This notebook exists to test & tune the core controller logic.
I included it so Jimmy & Karmesh could get insight into the controller behavior because we were debugging why the integrated version wasn't working back then.
I think we can remove the notebook now the continuous actions are working (and add tests to ensure that). @jimmytyyang @ykarmesh what do you guys think?

Reply:
Yeah I dont think I will need it. It will still be good to keep it somewhere internally maybe as a google colab? @vincentpierre Any thoughts?

Reply:
Code that we do not keep using will inevitably stop working after a few weeks.
I would prefer if this code was shortened, documented, tested and improved so that an external user could also benefit from it. If that is unrealistic, let's remove it.
If we want to keep it for internal reference for a whole, let's make it a gist or a colab somewhere and link to it in a code comment, but chances are it will be out of date very soon...

Reply:
Thank you, based on the above discussion, I will remove the file. Thank you everyone!